docs/develop-resources/README.md

```diff
@@ -18,8 +22,14 @@ There are definitely more work need to be done, but we'd like to share the steps
 
 ## Develop by scenraios
 
+You may only need to add a new field, or to promote an Alpha resource to Beta rather than writing an entirely new resource. To figure out the most suitable scenario, you need to know the resources current status:
+- [ ] Check the latest [CRD](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/crds). If the resource exists but in Alpha, you won’t see it in  [Config Connector API Reference](https://cloud.google.com/config-connector/docs/reference/overview) until it is promoted to Beta. Please follow [promote alpha to beta](./scenarios/alpha-to-beta.md) guide.
```

Comment:
These instructions are correct but assume a high level of familiarity. Maybe something more like the following?

Check the latest [CRD](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/crds) for the cnrm.cloud.google.com/stability-level annotation. The [Config Connector API Reference](https://cloud.google.com/config-connector/docs/reference/overview) only contains APIs of at least 'beta' stability. CRDs with 'alpha' stability need to be promoted to 'beta' stability to be formally documented in the API reference guide. Please follow the [promote alpha to beta](./scenarios/alpha-to-beta.md) guide.